config/locales/en/admin/whats_new.yml

```diff
@@ -12,6 +12,17 @@ en:
           This will make Whitehall Publisher more accessible, more user friendly and give you a consistent user experience throughout your publishing journey on GOV.UK.
 
           [Read more about the changes on the content community Basecamp](https://3.basecamp.com/4322319/buckets/15005645/messages/5287245970).
+      upcoming_changes:
+        heading: Upcoming changes
+        updates:
+          - heading: Reusing previous withdrawal dates and public explanations
+            area: Creating and updating documents
+            type: improved
+            date: 26 September 2022
+            body_govspeak: |
+              When re-withdrawing content, you can choose to reuse a previous withdrawal date and public explanation in certain circumstances.
+
+              [Read the guidance about when you can reuse previous withdrawal dates and explanations](https://www.gov.uk/guidance/how-to-publish-on-gov-uk/unpublishing-and-archiving).
```

Comment:
@johnpauldickie - added some placeholder content, could you let me know what to put here?